wdl-grammar/tests/parsing/name-ref/source.wdl

```diff
@@ -0,0 +1,9 @@
+# This is a test of a name references.
+
+version 1.1
+
+task test {
+    Integer x = 0
+    Integer y = -x
+    Integer z = x + y * x
```

Comment:
```suggestion
    Int x = 0
    Int y = -x
    Int z = x + y * x
```
I think this is what was meant. What is currently being parsed is a custom type named `Integer` (that doesn't exist).

Reply:
Doh! `Integer` is legal an would be interpreted as a custom type name here; this is going to be a recurring problem in my WDL journey, I think 😭